server/views/pages/curated-lists.njk

```diff
@@ -1,5 +1,9 @@
 {% extends 'layout/default.njk' %}
 
+{% block pageMeta %}
+  <meta name="description" content="Exploration for the incurably curious" />
```

Comment:
Will see what I can come up with. Is it specifically for articles lists as mentioned in #1099?